### PR TITLE
fixed some code smell

### PR DESCRIPTION
**Pull request by J3fftw1**
## Description
Changed some stuff to protected

## Changes
Changed some stuff to protected
Added some annotations

## Related Issues
N/A

## Checklist

- [x] I have fully tested the proposed changes and promise that they will not break everything into chaos.
- [x] I have also tested the proposed changes in combination with various popular addons and can confirm my changes do not break them.
- [ ] I followed the existing code standards and didn't mess up the formatting.
- [ ] I did my best to add documentation to any public classes or methods I added.
- [x] I have added `Nonnull` and `Nullable` annotations to my methods to indicate their behaviour for null values
- [ ] I added sufficient Unit Tests to cover my code.

### CODE REVIEW

**Comment by TheBusyBiscuit** [comment]
Please stop messing up the formatting.

File: src/main/java/io/github/thebusybiscuit/slimefun4/implementation/SlimefunPlugin.java [hunk 1]
```diff
@@ -1,27 +1,5 @@
...
-
-import org.bukkit.Bukkit;
-import org.bukkit.Server;
-import org.bukkit.command.Command;
-import org.bukkit.entity.Player;
-import org.bukkit.plugin.Plugin;
-import org.bukkit.plugin.PluginDescriptionFile;
-import org.bukkit.plugin.java.JavaPlugin;
-import org.bukkit.plugin.java.JavaPluginLoader;
-
```
Comment (LinoxGH) [inline src/main/java/io/github/thebusybiscuit/slimefun4/implementation/SlimefunPlugin.java:24]: Undo this.

File: src/main/java/io/github/thebusybiscuit/slimefun4/implementation/SlimefunPlugin.java [hunk 2]
```diff
@@ -198,15 +196,18 @@ else if (getServer().getPluginManager().isPluginEnabled("CS-CoreLib")) {
...
             // Setting up Networks
             gpsNetwork = new GPSNetwork();
 
             int networkSize = config.getInt("networks.max-size");
 
             if (networkSize < 1) {
-                getLogger().log(Level.WARNING, "Your 'networks.max-size' setting is misconfigured! It must be at least 1, it was set to: {0}", networkSize);
+                getLogger().log(Level.WARNING,
+                    "Your 'networks.max-size' setting is misconfigured! " +
+                        "It must be at least 1, it was set to: {0}", networkSize);
```
Comment (LinoxGH) [inline src/main/java/io/github/thebusybiscuit/slimefun4/implementation/SlimefunPlugin.java:210]: Indentation error.

File: src/main/java/io/github/thebusybiscuit/slimefun4/implementation/SlimefunPlugin.java [hunk 3]
```diff
@@ -251,7 +252,11 @@ else if (getServer().getPluginManager().isPluginEnabled("CS-CoreLib")) {
                     recipeService.refresh();
                 }
                 catch (Exception | LinkageError x) {
-                    getLogger().log(Level.SEVERE, x, () -> "An Exception occurred while iterating through the Recipe list on Minecraft Version " + minecraftVersion.getName() + " (Slimefun v" + getVersion() + ")");
+                    getLogger().log(Level.SEVERE, x, () ->
+                        "An Exception occurred while iterating through the Recipe list on " +
+                            "Minecraft Version " + minecraftVersion.getName() +
+                            " (Slimefun v" + getVersion() + ")"
```
Comment (LinoxGH) [inline src/main/java/io/github/thebusybiscuit/slimefun4/implementation/SlimefunPlugin.java:258]: Indentation error.

File: src/main/java/io/github/thebusybiscuit/slimefun4/implementation/SlimefunPlugin.java [hunk 4]
```diff
@@ -376,7 +389,10 @@ public void onDisable() {
                 entry.getValue().saveAndRemove();
             }
             catch (Exception x) {
-                getLogger().log(Level.SEVERE, x, () -> "An Error occurred while saving Slimefun-Blocks in World '" + entry.getKey() + "' for Slimefun " + getVersion());
+                getLogger().log(Level.SEVERE, x, () ->
+                    "An Error occurred while saving Slimefun-Blocks in World '"
+                        + entry.getKey() + "' for Slimefun " + getVersion()
```
Comment (LinoxGH) [inline src/main/java/io/github/thebusybiscuit/slimefun4/implementation/SlimefunPlugin.java:394]:
```suggestion
                    "An Error occurred while saving Slimefun-Blocks in World '" +
                    entry.getKey() + "' for Slimefun " + getVersion()
```

File: src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/armor/HazmatArmorPiece.java [hunk 5]
```diff
@@ -1,38 +1,39 @@
 package io.github.thebusybiscuit.slimefun4.implementation.items.armor;
 
-import org.bukkit.NamespacedKey;
-import org.bukkit.inventory.ItemStack;
-import org.bukkit.potion.PotionEffect;
-
-import io.github.thebusybiscuit.slimefun4.core.attributes.ProtectiveArmor;
```
Comment (LinoxGH) [inline src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/armor/HazmatArmorPiece.java:7]: Undo this replacement.

File: src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/armor/HazmatArmorPiece.java [hunk 6]
```diff
@@ -1,38 +1,39 @@
...
 
 /**
  * Represents 1 {@link SlimefunArmorPiece} of the Hazmat armor set.
  * One of the very few utilisations of {@link ProtectiveArmor}.
  *
  * @author Linox
- *
  * @see SlimefunArmorPiece
  * @see ProtectiveArmor
- *
```
Comment (LinoxGH) [inline src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/armor/HazmatArmorPiece.java:22]: Can you please stop messing the code format.

File: src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/armor/HazmatArmorPiece.java [hunk 7]
```diff
@@ -1,38 +1,39 @@
...
     private final NamespacedKey namespacedKey;
     private final ProtectionType[] types;
 
-    public HazmatArmorPiece(Category category, SlimefunItemStack item, RecipeType recipeType, ItemStack[] recipe, PotionEffect[] effects) {
+    public HazmatArmorPiece(Category category, SlimefunItemStack item,
+                            RecipeType recipeType, ItemStack[] recipe, PotionEffect[] effects) {
         super(category, item, recipeType, recipe, effects);
 
-        types = new ProtectionType[] { ProtectionType.BEES, ProtectionType.RADIATION };
+        types = new ProtectionType[] {ProtectionType.BEES, ProtectionType.RADIATION};
```
Comment (LinoxGH) [inline src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/armor/HazmatArmorPiece.java:32]: The spaces are there to clarify the values.

**Review by LinoxGH — CHANGES_REQUESTED** [review]

**Comment by LinoxGH** [comment]
Can you also **stop** splitting random lines? You are just making very little code parts take more lines than it needs.

**Comment by J3fftw1** [comment]
im making the code readable but sure will undo the changes

**Comment by LinoxGH** [comment]
> im making the code readable but sure will undo the changes

It just creates inconsistency and random indentation errors. And the new lines can easily be mistaken as part of a lambda or anything really.

**Comment by J3fftw1** [comment]
> 
> 
> > im making the code readable but sure will undo the changes
> 
> It just creates inconsistency and random indentation errors. And the new lines can easily be mistaken as part of a lambda or anything really.

ugh sure if you say so

**Comment by TheBusyBiscuit** [comment]
@J3fftw1 Follow the existing code format, making the code this inconsistent is not beneficial to anyone.
And what style of coding is understood to be the "most readable" is very subjective anyway.
Let's rather be consistent with this instead of the code looking like an inconsistent mess again.

Whether or not changing the access level of these fields is a desirable change anyway is still up for debate, I would have had preferred to wait for @WalshyDev for this to check since he added these fields and making them protected would rather go against the entire point of him adding these, so I'll still wait for him to review this.

**Review by LinoxGH — APPROVED** [review]
Btw if you are looking for changes to make for hacktoberfest, you can add `@Nonnull` and `@Nullable` annotations.

**Comment by J3fftw1** [comment]
i just need to remember to not hit ctrl alt l and i keep forgetting that but i will keep it in mind!

> Btw if you are looking for changes to make for hacktoberfest, you can add @Nonnull and @Nullable annotations.

already done with hacktober pr's but i wanna make 5 prs a week

**Comment by TheBusyBiscuit** [comment]
You can also configure the formatting settings on a per-project level, that's what I am doing when I work on other people's projects that use completely different formatting rules or bracket styles.

File: src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/armor/HazmatArmorPiece.java [hunk 8]
```diff
@@ -19,7 +21,7 @@
  *
  * @see SlimefunArmorPiece
  * @see ProtectiveArmor
- *
+ *
```
Comment (WalshyDev) [inline src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/armor/HazmatArmorPiece.java:24]: Added whitespace, remove

File: src/main/java/io/github/thebusybiscuit/slimefun4/core/services/localization/SupportedLanguage.java [hunk 9]
```diff
@@ -58,7 +58,7 @@
     MACEDONIAN("mk", false, "a0e0b0b5d87a855466980a101a757bcdb5f77d9f7287889f3efa998ee0472fc0"),
     TAGALOG("tl", true, "9306c0c1ce6a9c61bb42a572c49e6d0ed20e0e6b3d122cc64c339cbf78e9e937");
 
-    public static final SupportedLanguage[] values = values();
+    protected static final SupportedLanguage[] values = values();
```
Comment (WalshyDev) [inline src/main/java/io/github/thebusybiscuit/slimefun4/core/services/localization/SupportedLanguage.java:61]: This shouldn't be protected. The point of these values is to replace `values()` and save a ton of time.

Sonar may complain but that's due to it not being able see the use and addon usages. There's a comment to disable Sonar on a line, Google it

File: src/main/java/io/github/thebusybiscuit/slimefun4/core/services/profiler/PerformanceRating.java [hunk 10]
```diff
@@ -31,7 +31,7 @@
     HURTFUL(ChatColor.DARK_RED, 500),
     BAD(ChatColor.DARK_RED, Float.MAX_VALUE);
 
-    public static final PerformanceRating[] values = values();
+    protected static final PerformanceRating[] values = values();
```
Comment (WalshyDev) [inline src/main/java/io/github/thebusybiscuit/slimefun4/core/services/profiler/PerformanceRating.java:34]: Same as above

**Review by WalshyDev — CHANGES_REQUESTED** [review]

File: src/main/java/io/github/thebusybiscuit/slimefun4/core/services/localization/SupportedLanguage.java (hunk 9 above)
Comment (J3fftw1) [reply src/main/java/io/github/thebusybiscuit/slimefun4/core/services/localization/SupportedLanguage.java:61]: i found a way to disable it but will this for just me or for everyone else?

**Review by LinoxGH — APPROVED** [review]

**Pull request closed without merging**